openwisp_network_topology/tests/test_selenium.py

```diff
@@ -151,3 +153,38 @@ def test_topology_non_admin_view_graph_visualizer(self):
         )
         topology_graph_element.click()
         self._assert_topology_graph()
+
+    def test_topology_admin_visualizer_multiple_close_btn_append(self):
```

Comment:
is this test related to this PR or another one?

Reply:
This is related to the `"closeBtn"` PR. I have rebased this branch on the latest master. :+1: